app/keepers/keepers.go

```diff
@@ -508,10 +535,65 @@ func (ak *AppKeepers) InitKeepers(
 		runtime.NewKVStoreService(keys[finalitytypes.StoreKey]),
 		ak.BTCStakingKeeper,
 		ak.IncentiveKeeper,
-		ak.CheckpointingKeeper,
+		&checkpointingKeeper,
 		appparams.AccGov.String(),
 	)
 
+	monitorKeeper := monitorkeeper.NewKeeper(
+		appCodec,
+		runtime.NewKVStoreService(keys[monitortypes.StoreKey]),
+		&btclightclientKeeper,
+	)
+
+	// create querier for KVStore
+	storeQuerier, ok := bApp.CommitMultiStore().(storetypes.Queryable)
```

Comment:
it seems out of scope to get the store query root into a keeper, and it seems like the zone concierge already imports the needed keeper epoching and checkpointing

Reply:
This is necesasry to get the inclusion proof of some metadata in BTC timestamping, see [here](https://github.com/babylonlabs-io/babylon/blob/873f1232365573a97032037af4ac99b5e3fcada8/x/zoneconcierge/keeper/query_kvstore.go#L19-L20).